src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -211,221 +204,10 @@ def list_jobs(self):
             kblogging.log_event(self._log, "list_jobs.error", {"err": str(e)})
             raise
 
-    def _create_error_state(
-        self,
-        error: str,
-        error_msg: str,
-        code: int,
-        cell_id=None,
-        run_id=None,
-        job_id=None,
-    ) -> dict:
-        """
-        Creates an error state to return if
-        1. the state is missing or unretrievable
-        2. Job is none
-        This creates the whole state dictionary to return, as described in
-        _construct_job_status.
-        :param error: the full, detailed error (not necessarily human-readable, maybe a stacktrace)
-        :param error_msg: a shortened error string, meant to be human-readable
-        :param code: int, an error code
-        """
-        return {
-            "status": "error",
-            "error": {
-                "code": code,
-                "name": "Job Error",
-                "message": error_msg,
-                "error": error,
-            },
-            "errormsg": error_msg,
-            "error_code": code,
-            "job_id": job_id,
-            "cell_id": cell_id,
-            "run_id": run_id,
-            "created": 0,
-            "updated": 0,
-        }
-
-    def _construct_job_status(self, job: Job, state: dict) -> dict:
```

Comment:
this method is now part of the Job object, correct?

Reply:
Yeah, I changed the name to job.revised_state